resources/server/sv_twitter.ts

```diff
@@ -132,8 +133,12 @@ async function createTweet(identifier: string, tweet: NewTweet): Promise<Tweet>
     tweet.images,
     tweet.retweet,
   ]);
+  // This should not be an any type and instead should be
+  // a Tweet[] according to mysql2 documentation. But instead
+  // this exec promise returns the row itself, not an array of rows like it states
+  // Therefore type assertion doesn't work here for correctly typing this out.
   const insertData = <any>results;
-  return await getTweet(profile.id, insertData.insertId);
+  return await getTweet(profile.id, insertData.id);
```

Comment:
@TasoOneAsia this is the line I was mainly comenting on, I don't think `id` exists? `insertId` is changed here.

Reply:
Fixed yea, I had forgotten I had changed that a couple days ago, should be resolved now with 357acd4